[venus-worker] vc-processors 导出了不存在的函数

### 模块 / Components

- [ ] venus-sector-manager
- [X] venus-worker
- [ ] 工具链 / toolchains
- [ ] 文档 / docs

### 版本 / Version

```text
vc-processors main 分支
```


### 描述 / Describe the Bug

依赖 vc-processors main 分支的版本会报错

error:
```
error[E0432]: unresolved import `storage_proofs_porep::stacked::init_numa_mem_pool`
  --> /.../.cargo/git/checkouts/venus-cluster-a7a3193dc459062c/bd69e5d/venus-worker/vc-processors/src/fil_proofs/mod.rs:33:9
   |
33 | pub use storage_proofs_porep::stacked::init_numa_mem_pool;
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `init_numa_mem_pool` in `stacked`

For more information about this error, try `rustc --explain E0432`.
error: could not compile `vc-processors` due to previous error
```

### 日志 / Logging Information

```text
no
```


### 重现步骤 / Repo Steps

_No response_

--- a/venus-worker/Cargo.toml
+++ b/venus-worker/Cargo.toml
@@ -81,6 +81,10 @@ vc-processors = { path = "./vc-processors" }
 [target.'cfg(not(target_os = "macos"))'.dependencies]
 vc-processors = { path = "./vc-processors", features = ["numa"] }
 
+[dependencies.storage-proofs-porep]
+version = "11.1"
+default-features = false
+
 [dev-dependencies]
 pretty_assertions = "1.2"
 humantime = "2.1"

--- a/venus-worker/Cargo.toml
+++ b/venus-worker/Cargo.toml
@@ -82,7 +82,9 @@ vc-processors = { path = "./vc-processors" }
 vc-processors = { path = "./vc-processors", features = ["numa"] }
 
 [dependencies.storage-proofs-porep]
-version = "11.1"
+git = "https://github.com/ipfs-force-community/rust-fil-proofs"
+package = "storage-proofs-porep"
+branch = "force/master_v11.1.1"
 default-features = false
 
 [dev-dependencies]

--- a/venus-worker/src/bin/venus-worker/processor/mod.rs
+++ b/venus-worker/src/bin/venus-worker/processor/mod.rs
@@ -78,7 +78,7 @@ pub(crate) fn submatch(subargs: &ArgMatches<'_>) -> Result<()> {
         (STAGE_NAME_ADD_PIECES, _) => run_consumer::<AddPieces, BuiltinProcessor>(),
 
         (STAGE_NAME_PC1, Some(m)) => {
-            use vc_processors::fil_proofs::init_numa_mem_pool;
+            use storage_proofs_porep::stacked::init_numa_mem_pool;
             use venus_worker::seal_util::{scan_memory_files, MemoryFileDirPattern};
 
             // Argument `hugepage_files_path_pattern` take precedence over argument `hugepage_files_path`

--- a/venus-worker/vc-processors/src/fil_proofs/mod.rs
+++ b/venus-worker/vc-processors/src/fil_proofs/mod.rs
@@ -30,7 +30,6 @@ pub use filecoin_proofs_api::{
     ChallengeSeed, Commitment, PaddedBytesAmount, PartitionProofBytes, PieceInfo, PrivateReplicaInfo, ProverId, RegisteredPoStProof,
     RegisteredSealProof, RegisteredUpdateProof, SectorId, Ticket, UnpaddedBytesAmount,
 };
-pub use storage_proofs_porep::stacked::init_numa_mem_pool;
 
 /// Identifier for Actors.
 pub type ActorID = u64;

